components/annot-missing-values.vue

```diff
@@ -68,133 +65,41 @@
-                    for ( const value of this.uniqueValues[columnName] ) {
+                "missingColumnValues"
+            ]),
 
-                        switch ( this.dataType ) {
+            tableItems() {
+                let missingValueArray = []
 
-                            case "categorical":
-                                if ( !this.isValidCategorical(columnName, value) ) {
-                                    missingValuesLists[columnName].push(value);
+                for ( let column of this.relevantColumns ) {
+                    if ( Object.keys(this.missingColumnValues).includes(column) ) {
+                        for ( let missing_value of this.missingColumnValues[column] ) {
+                            const description = this.valueDescription(column, missing_value);
+                            missingValueArray.push(
+                                {
+                                    column: column,
+                                    description: description === null ? "" : description,
+                                    value: missing_value
                                 }
-                                break;
+                            )
 
-                            case "continuous":
-                                if ( !this.isValidContinuous(value) ) {
-                                    missingValuesLists[columnName].push(value);
-                                }
-                                break;
-
-                            case "string":
-                                if ( !this.isValidString(value) ) {
-                                    missingValuesLists[columnName].push(value);
-                                }
-                                break;
                         }
                     }
                 }
+                return missingValueArray
+            }
+        },
 
-                // 2. Save the missing values lists to the store
-                this.$emit("update:missingColumnValues", missingValuesLists);
-            },
-
-            isValidCategorical(p_column, p_value) {
-
-                let isValid = true;
-
-                // 1. Value is potentially invalid if it is missing from the data dictionary description
-                if ( null === this.valueDescription(p_column, p_value) ) {
-                    isValid = false;
-                }
-
-                return isValid;
-            },
-
-            isValidContinuous(p_value) {
-
-                let isValid = true;
-
-                // 1. Value is likely invalid if it is not numeric
-                if ( isNaN(p_value) ) {
-                    isValid = false;
-                }
-
-                return isValid;
-            },
-
-            isValidString(p_value) {
-
-                let isValid = true;
-
-                return isValid;
-            },
+        methods: {
 
             removeColumn(p_tableItem) {
```

Comment:
Check for trailing whitespace in this function

Reply:
Thanks, but I don't see any trailing whitespace in this method. Do you?

Reply:
The linter picked up whitespace at lines 193 and 197, yes.